### DISCUSSION TITLE
19043.1165 wslg?

### DISCUSSION THREAD

**Discussion by faustort** [microsoft/wslg#413] [Q&A] [answered]
I'm stuck with Windows 10 compilation 19043.1165, and there's no sign of WSLg work on it. 
I was using Windows 11 on Dev Channel, but I had to downgrade to Windows 10 and now I can't switch to DEV channel again as my PC is not compatible with W11.

I've tried to disable WSL and enable it again... and lot's of other things, end up I'm figuring that there was no server listening to GUI.

Can someone give me a light on this please?

Windows 10 Pro
21H1
‎13/‎08/‎2021
19043.1165
Windows Feature Experience Pack 120.2212.3530.0

**Comment by crramirez** [comment, marked as answer]
There are no plans to backport WSLg to Windows 10.

You should be able to switch to DEV channel nor Beta even if your PC is not compatible.

**Reply by faustort** [reply]
Thanks for elucidate that @crramirez
